xml/System/Half.xml

```diff
@@ -236,9 +238,10 @@
         <Parameter Name="value" Type="System.Half" />
       </Parameters>
       <Docs>
-        <param name="value">To be added.</param>
-        <summary>Determines whether the specified value is infinite.</summary>
-        <returns>To be added.</returns>
+        <param name="value">A <seealso cref="T:System.Half" /> floating-point number.</param>
+        <summary>Returns a value indicating whether the specified number evaluates to positive infinity.</summary>
+        <returns>
+          <see langword="true" /> if <paramref name="value" /> evaluates to <seealso cref="P:System.Half.PositiveInfinity" />; otherwise <see langword="false" />.</returns>
```

Comment:
```suggestion
          <see langword="true" /> if <paramref name="value" /> evaluates to <see cref="P:System.Half.PositiveInfinity" />; otherwise <see langword="false" />.</returns>
```